### PR TITLE
[VEN-2954] [VEN-2955] Deploy XVS and XVS Bridge on Base Mainnet

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -23,3 +23,4 @@ DEPLOYER_PRIVATE_KEY=
 #ARCHIVE_NODE_opsepolia=https://sepolia.optimism.io
 #ARCHIVE_NODE_opmainnet=https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_basesepolia=https://sepolia.base.org
+#ARCHIVE_NODE_basemainnet=https://mainnet.base.org
```
Comment (chechu) [inline .env.example:26]:
```suggestion
#ARCHIVE_NODE_basemainnet=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/base
```

**Review by chechu — APPROVED** [review]

File: hardhat.config.ts [hunk 2]
```diff
@@ -184,6 +188,12 @@ const config: HardhatUserConfig = {
       live: true,
       accounts: DEPLOYER_PRIVATE_KEY ? [`0x${DEPLOYER_PRIVATE_KEY}`] : [],
     },
+    basemainnet: {
+      url: process.env.ARCHIVE_NODE_basemainnet || "https://mainnet.base.org",
+      chainId: 8453,
+      live: true,
+      accounts: process.env.DEPLOYER_PRIVATE_KEY ? [`0x${process.env.DEPLOYER_PRIVATE_KEY}`] : [],
```
Comment (GitGuru7) [inline hardhat.config.ts:195]:
```suggestion
    accounts: DEPLOYER_PRIVATE_KEY ? [`0x${DEPLOYER_PRIVATE_KEY}`] : [],
```

For consistency

**Review by GitGuru7 — APPROVED** [review]

File: hardhat.config.ts [hunk 3]
```diff
@@ -63,6 +63,10 @@ extendConfig((config: HardhatConfig) => {
           "node_modules/@venusprotocol/governance-contracts/deployments/basesepolia",
           "node_modules/@venusprotocol/oracle/deployments/basesepolia",
         ],
+        basemainnet: [
+          // TODO: "node_modules/@venusprotocol/governance-contracts/deployments/basemainnet",
+          // TODO: "node_modules/@venusprotocol/oracle/deployments/basemainnet",
+        ],
```
Comment (kkirka) [inline hardhat.config.ts:69]: Why is it a todo?

File: hardhat.config.ts (hunk 2 above)
Comment (chechu) [reply hardhat.config.ts]: https://github.com/VenusProtocol/token-bridge/pull/45/commits/9c8e6a7250087dc9dfccd5303f6266b6896d6dcc

File: hardhat.config.ts (hunk 3 above)
Comment (chechu) [reply hardhat.config.ts]: https://github.com/VenusProtocol/token-bridge/pull/45/commits/ba216c4e20ea6fff9d581a4cfbefbe9a765cde9f

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**